.mailmap

```diff
@@ -28,6 +28,7 @@ Mark Worsfold <mark.worsfold@metoffice.gov.uk> <35810969+MarkWorsfold@users.nore
 Meabh NicGuidhir <meabh.nicguidhir@metoffice.gov.uk> <meabh.nicguidhir@metoffice.gov.uk>
 Neil Crosswaite <neil.crosswaite@metoffice.gov.uk> <43375279+neilCrosswaite@users.noreply.github.com>
 Paul Abernethy <paul.abernethy@metoffice.gov.uk> <paul.abernethy@metoffice.gov.uk>
+Peter Jordan <peter.jordan@metoffice.gov.uk> <peter.jordan@metoffice.gov.uk>
```

Comment:
This might help, but I still think you need to re-commit the existing `peter.jordan` commits as `MO-PeterJordan`
```suggestion
Peter Jordan <peter.jordan@metoffice.gov.uk> <52462411+mo-peterjordan@users.noreply.github.com>
```